reference-tests/src/test/java/net/consensys/linea/BlockchainReferenceTestTools.java

```diff
@@ -222,9 +240,10 @@ public static void executeTest(final BlockchainReferenceTestCaseSpec spec) {
 
     zkTracer.traceEndConflation(worldState);
 
-    ExecutionEnvironment.checkTracer(zkTracer, CORSET_VALIDATOR, Optional.of(log));
-
-    assertThat(blockchain.getChainHeadHash()).isEqualTo(spec.getLastBlockHash());
+    String result = ExecutionEnvironment.checkTracer(zkTracer, CORSET_VALIDATOR, Optional.of(log));
```

Comment:
If corset validation fails the checkTracer method will throw an assertion error and will never return the result of invalid trace validation, so this might not do what you inted.